src/cli/main.c

```diff
@@ -68,7 +68,7 @@ static void usage(const char *progname)
   fprintf(stderr, "   --export_masks <0|1|false|true>, default: false\n");
   fprintf(stderr, "   --style <style name>\n");
   fprintf(stderr, "   --style-overwrite\n");
-  fprintf(stderr, "   --apply-custom-presets <0|1|false|true>, default: true\n");
+  fprintf(stderr, "   --apply-custom-presets <0|1|false|true>, default: true, disable for multiple instances\n");
```

Comment:
This will make line bigger than 80 chars. I will put this message in a separate line (see -d option in darktable main executable).

Reply:
Any chance to have this for 3.2 ?

Reply:
Are you asking me? You said you'll put the message in a separate line so I thought you were going to add a commit to the PR. Do you want me to do it?

Reply:
Sorry I meant to say "I would put..." and not I will :) Yes, please do so. Thanks.

Reply:
Done but not sure about the indentation, let me know if you had something else in mind.